packages/mongo-nav/src/mongo-select/ProjectSelect.tsx

```diff
@@ -266,11 +268,18 @@ function ProjectSelect({
         >
           {current?.projectName ?? ''}
         </span>
-        <Icon
-          size="small"
-          glyph={open ? 'CaretUp' : 'CaretDown'}
-          className={cx(caretBaseStyle, { [iconLoadingStyle]: loading })}
-        />
+
+        {open ? (
+          <CaretUpIcon
+            size={Size.Small}
+            className={cx(caretBaseStyle, { [iconLoadingStyle]: loading })}
+          />
+        ) : (
+          <CaretDownIcon
+            size={Size.Small}
+            className={cx(caretBaseStyle, { [iconLoadingStyle]: loading })}
+          />
+        )}
```

Comment:
These ternaries inline feel a bit worse than doing something like:

```js
const CurrentIcon = open ? CaretUpIcon : CaretDownIcon;

...
<CurrentIcon size='small' className={cx(caretBaseStyle, { [iconLoadingStyle]: loading })} /?
```

I don't feel that strongly about it if you do though.

Reply:
👍